src/app/components/inputScreen/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const Container = styled.div`
+  flex: 1 1 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+`;
+
+const ButtonsContainer = styled.div`
+  margin: ${(props) => props.theme.spacing(12)}px 0;
+`;
+
+const InputsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  // gap: ${(props) => props.theme.space.xl};
+`;
+
+type InputScreenProps = {
+  inputs: React.ReactNode;
+  buttons: React.ReactNode;
+  header?: React.ReactNode;
+};
+
+function InputScreen({ inputs, buttons, header }: InputScreenProps) {
```

Comment:
I'm not 100% convinced this component is needed. It's currently used in 1 place with a single input and single button. Do we foresee it being used elsewhere?

Reply:
I kinda agree with @victorkirov here. I get the intent is to abstract away some of the layout logic to make it consistent, but IMO a better abstraction is to store well-defined styled components into ui-library/common.styled.ts.

because in this case, i'm not sure if this abstracted layout will work for stx with the memo input.